ic-helpers/src/factory/state.rs

```diff
@@ -23,11 +23,11 @@ macro_rules! impl_factory_state_management {
     ( $state:ident, $bytecode:expr ) => {
         impl $state {
             pub fn stable_save(&self) {
-                ::ic_cdk::storage::stable_save((self,)).unwrap();
+                ::ic_kit::ic::stable_store((self,)).unwrap();
```

Comment:
Can we get rid of these unwraps. Either use `expect` if it can't fail, otherwise we should return a `Result` here

Reply:
Sure we can. My reasoning was that if they're present, then this was intentional. From my perspective, ic canisters are prone to use unwraps instead of a proper error handling because I've been seeing this everywhere with the caption that it allows to kill the thread and revert the canister state.

Reply:
Also imo this macro is unneeded in first place because this separation of the state api between the crates is more confusing and probably unnecessary due to the way `ic-canister` canisters handle state

Reply:
probably is no longer needed..